openapi/openapi.yaml

```diff
@@ -1233,6 +1233,303 @@ paths:
+          in: query
+          required: false
+          description: 'Determines the end date of the period to select ads with activity for. Should represent a date equal to or after the ``startDate``.'
+          schema:
+            type: string
+            format: date
+          example: '2023-06-13'
+        - name: categoryIds
+          in: query
+          required: false
+          description: 'List of category IDs to filter by. Only leaf category id’s are useful, since ads can only be placed in leaf categories.'
+          schema:
+            type: array
+            items:
+              type: integer
+              format: int64
+          example: [ 55500, 1280000 ]
+        - name: campaignId
+          in: query
+          required: false
+          description: 'Campaign to filter ads by.'
+          schema:
+            type: integer
+            format: int64
+          example: 124422
+      responses:
+        '200':
+          description: 'Success'
+          headers:
+            Oauth-Scope:
+              description: 'Oauth scopes that are available with the supplied token'
+              schema:
+                type: string
+                example: 'api_ro api_rw'
+            Oauth-Scope-Required:
+              description: 'The required permission for this particular call'
+              schema:
+                type: string
+                example: 'get_ads'
+          content:
+            application/sellside.ad.count-v1+json:
+              schema:
+                type: integer
+                example: 124
+        '401':
+          description: 'Unauthorized'
+          content:
+            application/sellside.error-v1+json; charset=UTF-8:
+              schema:
+                type: array
+                items:
+                  $ref: './errors.yaml#/components/schemas/ApiResponseError'
+              example:
+                - code: 1009
+                  text: 'missing token'
+                  msg: 'Authorization header is missing or in the wrong format'
+        '406':
+          description: 'Not Acceptable'
+          content:
+            application/json:
+              schema:
+                $ref: './errors.yaml#/components/schemas/NotSupportedError'
+        '500':
+          description: 'Server Error'
+          content:
+            application/sellside.error-v1+json; charset=UTF-8:
+              schema:
+                type: array
+                items:
+                  $ref: './errors.yaml#/components/schemas/ApiResponseError'
+              example:
+                - code: 1000
+                  text: 'internal error'
+                  msg: 'The server encountered an error'
+      security:
+        - bearerAuth: [ ]
+          oAuth2ClientCredentials:
+            - 'console_ro'
+            - 'api_ro'
+  /ad/histogram:
+    get:
+      tags:
+        - Ads
+      summary: 'Get a histogram of ads with filter criteria, grouped by a given field'
```

Comment:
worth noting that only ads with activity are considered?

Reply:
only if timeframe is selected, yes, I'll add that.